syntax/std_fmt.go

```diff
@@ -15,8 +13,7 @@ func stdFmt() rel.Attr {
 				return nil, err
 			}
 
-			fmt.Println(prettifiedString)
-			return value, nil
+			return rel.NewString([]rune(prettifiedString)), nil
```

Comment:
Now its output is:
```
@> //fmt.pretty([1,2,3])
'[\n  1,\n  2,\n  3\n]'
@>  
```
It is not prettified.

Reply:
That's expected per Marcelo's comment.